airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/RedshiftStagingS3DestinationAcceptanceTest.java

```diff
@@ -52,8 +53,8 @@ protected JsonNode getConfig() {
     return config;
   }
 
-  public JsonNode getStaticConfig() {
-    return Jsons.deserialize(IOs.readFile(Path.of("secrets/config.json")));
+  public JsonNode getStaticConfig() throws IOException {
+    return Jsons.deserialize(IOs.readFile(Path.of("secrets/config_staging.json")));
```

Comment:
please mention which creds should be placed as `secrets/config_staging.json` in the readme file